server/modules/index.js

```diff
@@ -2,91 +2,91 @@ module.exports = [
     {
         name: 'util.tokenGenerator',
         dependencies: [],
-        require: __dirname + '/token-generator'
+        module: require(__dirname + '/token-generator')
```

Comment:
Yeah, you're right. Probably I was thinking about making it lazy at some point but forgot about it. Anyway, as I don't think lazy loading would have any impact on performance here, I'm fine with this change.

I have no opinion about ES module loading though.